examples/resources/trocco_connection/google_spreadsheets.tf

```diff
@@ -0,0 +1,14 @@
+resource "trocco_connection" "google_spreadsheets" {
```

Comment:
Please insert to this example code to `templates/resources/connection.md.tmpl` ;)